File: python/PatternPrinting/Alphabet_pyramid.py
#x=int(input("enter a number:"))
#a=65
#for i in range(1,x+1):
   # print("  " * (x-i), end=' ')
    #for j in range(1,2*i):
     #   print(chr(a),end=' ')
      #  a+=1
    #print()

   
   
x=int(input("enter a number:"))

for i in range(1,x+1):
    print("  " * (x-i), end=' ')
    for j in range(65,2*i+64):
        print(chr(j),end=' ')
        
    print()

File: c++/Alphabet_Pyramid.cpp
#include<iostream>
using namespace std;
int main(){
     int n;
    cout<<"enter a number:";
    cin>>n;
    int a=1;
    for(int i=1;i<=n;i++){
        
        for(int j=1;j<=n-i;j++){
            cout<<"  ";
        }
         for(int j=1;j<=a;j++){
            cout<<(char)(j+64)<<" ";
            
        }
         cout<<"\n";
         a+=2;
    }
}
   